app/gate_io.py

```diff
@@ -1,74 +1,117 @@
-import gate_api
-
 import gate_api
 from gate_api.exceptions import ApiException, GateApiException
+import csv
+import pytz
```

Comment:
did u pip install this? both pytz and csv? if so, could u add them to requirements.txt

Reply:
@walimike Just did pip freeze and `pytz` got added, but csv should already be part of python